pkg/transfer/transfer.go

```diff
@@ -86,8 +87,7 @@ type ImageExportStreamer interface {
 }
 
 type ImageUnpacker interface {
-	// TODO: consider using unpack options
-	UnpackPlatforms() []unpack.Platform
+	UnpackPlatforms() []image.UnpackConfiguration
```

Comment:
`UnpackConfiguration` should move to this package if it is used here. Loading from a subpackage could create an import loop (which maybe was the reason you ended up moving registry stuff).

Reply:
Yeah, I had cycle import and ended up create new package for registry as it has dependency on its parent transfer package.
I moved  `UnpackConfiguration` up per your suggestion.